packages/esm-patient-orders-app/src/routes.json

```diff
@@ -24,8 +24,10 @@
     {
       "name": "order-basket-action-menu",
       "component": "orderBasketActionMenu",
-      "slot": "action-menu-patient-chart-items-slot",
-      "order": 0
+      "slots": [
+        "action-menu-patient-chart-items-slot", 
+        "action-menu-ward-patient-items-slot"
```

Comment:
Can we also add this extension to the slot via config please?

Reply:
Updated the config here: https://github.com/openmrs/openmrs-distro-referenceapplication/pull/876/files